waspc/src/Wasp/Node/Version.hs

```diff
@@ -16,26 +13,53 @@ import qualified Text.Regex.TDFA as R
 import qualified Wasp.SemanticVersion as SV
 import Wasp.Util (indent)
 
+-- | Wasp supports any node version equal or greater to this version.
+-- | We usually keep this one equal to the latest LTS.
+oldestWaspSupportedNodeVersion :: SV.Version
+oldestWaspSupportedNodeVersion = SV.Version 20 0 0
```

Comment:
I have no problem keeping this as is, the ordering doesn't matter that much. Still, I'll give my reasoning why I disagree for the sake of argument (but don't feel like you need to change anything).

Each exported symbol is a story of its own. The file's name is `Version`, not `oldestNodeVersionThatWaspSupports`. Therefore, I'd argue that all exported symbols are equally important.

Viewing `oldestWaspSupportedNodeVersion` as more important than the `VersionCheckResult` (or something else) is IMO arbitrary/subjective and depends where you're coming from. I don't think its perceived importance justifies breaking the convention.

Reply:
TLDR; I disagree, telling a story (when possible) is important, order is a big part of it, and there is nothing wrong with the story being subjective.

Read on for more details:

> I don't think its perceived importance justifies breaking the convention.
-> Our convention is ordering code like a story, top to bottom.

> Therefore, I'd argue that all exported symbols are equally important.
Based on what? No, they are not. If each symbol is story of its own, then there is no story. Yes, maybe that is true in Util.hs file, but not in this file. I wrote the code here with the idea of how the stuff in this file relates with each other, and I ordered them in such a way that for you, the reader, reading will make sense.

Trying to say that there is no real importance sounds nice theoretically, and for the sake of the argument, but in practice it doesn't make much sense -> there is hierarchy, there is importance, and there is a story to be told in this file, and you could call it subjective, it is in a way, because the author of the code subjectively came up with it, but from that moment on, it is objectively what the author wanted you to read.